### ISSUE TITLE
Enhancement: ignore blocks on schematic

### ISSUE THREAD

**Issue by soepie7** [Lunatrius/Schematica#324]
I thought it would be nice to allow Schematica to ignore a certain block specified so that it will not check if it is the correct block, and will not be marked with any color.

This can be useful for schematics made for underground places, so that you won't get markings for all the "wrong" blocks like andesite, dirt or cave systems. 

Without this the materials tab is not as effective on underground schematics. It would also reduce the amount of markings rendered, which can improve fps on bigger schematics.

**Comment by Lunatrius** [comment]
You can already add blocks as air blocks in the config, but use that setting with caution, because it might make things behave in a weird way in regards to the printer.

**Comment by soepie7** [comment]
But blocks as air still give that purple marking if another block is placed, do they not?

**Comment by Lunatrius** [comment]
Yeah, that's what would happen... I'll see if anything can be done about it. I'm open to any suggestions that you might have about this.

**Comment by KrisBigK** [comment]
IMO that should definitely be added, as I have a similar idea and it would be quite helpful. For example in creative my contraptions are built in all kinds of concrete to mark out circuits with different purpose, but in survival I simply want to build them in cobblestone. This is when ignoring certain blocks can be handy. Ignoring the selected concrete blocks will reduce confusion greatly.
